assets/Product.wxs

```diff
@@ -59,7 +59,7 @@
       <Publish Dialog="MyExitDialog" Control="Finish" Order="1" Event="DoAction" Value="LaunchApplication">LAUNCHAPPONEXIT</Publish>
     </UI>
     <!-- Prerequisites -->
-    <Condition Message="Supported only on Win8 and above">
+    <Condition Message="Supported only on Win7 and above">
```

Comment:
Very minor but should we maybe spell out `Win7` completely as `Windows 7` and add a full stop?

Reply:
Yes, we should spell it out completely

Reply:
Fixed.